From: nabijaczleweli <nabijaczleweli@gmail.com>
Date: Mon, 6 Jan 2020 07:37:19 +0100
Subject: Fix lack of soname by setting BLT's VERSION

---
 Terminal/CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Terminal/CMakeLists.txt b/Terminal/CMakeLists.txt
index 20e3ac0..bac2887 100644
--- a/Terminal/CMakeLists.txt
+++ b/Terminal/CMakeLists.txt
@@ -77,6 +77,9 @@ elseif(APPLE)
 	target_link_libraries(BearLibTerminal ${COCOA_LIBRARY})
 endif()
 
+set_target_properties(BearLibTerminal PROPERTIES
+	VERSION ${TERMINAL_VERSION})
+
 if (WIN32)
 	# Fix unix-like "lib" prefix and ".a" suffix
 	set_target_properties(BearLibTerminal PROPERTIES
